crates/medea_derives/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "medea-derives"
```

Comment:
It's better to name `medea_macro`, as may hold not derive macros only (in future).